inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/adapter/RecommendationSpanRenderer.java

```diff
@@ -183,8 +189,13 @@ public void render(JCas aJcas, VDocument vdoc, AnnotatorState aState,
                     AnnotationObject ao = confidencePerClassifier.get(recommenderId);
 
                     if (first) {
-                        HashMap<String, String> featureAnnotation = new HashMap<String, String>();
-                        featureAnnotation.put(ao.getFeature(), ao.getAnnotation());
+                        AnnotationFeature feature = aAnnotationService
+                            .getFeature(ao.getFeature(), layer);
+                        String annotation = defaultString(aFsRegistry.getFeatureSupport(feature)
```

Comment:
What is happening here. Please add a comment explaining why defaultString is being used here.